tests/test_cli.py

```diff
@@ -204,6 +204,7 @@ def test_command_exits_with_positive_return_code_when_issues_are_found(
                 types.IssueType.Entropy, "foo", types.Chunk("foo", "/bar", {})
             )
         ]
+        mock_scanner.return_value._issue_count = 1  # pylint: disable=protected-access
```

Comment:
This could be changed to use the non-protected attribute like the others in this file